WebSite/Redirect.aspx

```diff
@@ -15,8 +15,7 @@
 
             <div class="panel-body">
                 <a href="Redirect.aspx?redirectType=ResponseRedirect&value=default.aspx">Redirect to default.aspx with Response redirect</a> <br/>
-                <a href="Redirect.aspx?redirectType=HeaderRedirect&value=default.aspx">Redirect to default.aspx with header and return code</a> <br/>
-                <a href="Redirect.aspx?redirectType=ResponseRedirectLocation&value=default.aspx">Redirect to default.aspx with response redirect location</a>
+                <a href="Redirect.aspx?redirectType=HeaderRedirect&value=default.aspx">Redirect to default.aspx with header and return code</a>
```

Comment:
Reducing the links displayed in the Redirect page from 3 to 2. Keeping "ResponseRedirect" and "HeaderRedirect". The "ResponseRedirectLocation" test case may require browser cache to be cleared prior to test, so taking it out from the UI. Of course, it can still be exercised by the user if required, and can still be exercised by the test framework.